### PR TITLE
fix(STN-434): timing issue with .hb-stretch-vertical-linked-cards

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
Implements both a timeout and promise behavior to try to ensure the stretch cards load correctly. This may still not fix the issue on extremely slow connections.

## Steps to Test
1. `npm run test` passes
2. see the Jira issue here: https://sparkbox.atlassian.net/browse/STN-434 - watch the video and see if you can still replicate the bug!

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/js/equal-height-grid/equal-height-grid.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const equalHeightGrid = (element, specialWrapperClass) => {
       // If there is only 1 unique value, then all elements are the same height,
       // and in that case, we don't need to change the height at all
       if (uniqueHeights.length > 1) {
-        setTimeout(() => {
+        return new Promise((resolve, reject) => {
```
Comment (rkelbel) [inline docroot/themes/humsci/humsci_basic/src/js/equal-height-grid/equal-height-grid.js:28]: Nice use of a Promise!

File: docroot/themes/humsci/humsci_basic/src/js/equal-height-grid/index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import equalHeightGrid from './equal-height-grid';
 
+// Wait a 1 sec for page to load in before setting heights
```
Comment (rkelbel) [inline docroot/themes/humsci/humsci_basic/src/js/equal-height-grid/index.js:3]: Thank you for updating the comment to what's going on here.

**Comment by rkelbel** [comment]
I was unable to re-create the bug in the video, even trying out "low-end mobile" throttling in Google Chrome Dev Tools, there was a small page load time where the issue was present but eventually cleared up once the page fully loaded. 
Looks like this should fix this for most situations. Nice work!

**Review by rkelbel — APPROVED** [review]
These changes look good!
Nice job!

**Comment by silverli** [comment]
@pookmish do you mind if we merge this into the release branch as well since it's still open?

**Pull request merged**